sdks/python/apache_beam/runners/interactive/dataproc/dataproc_cluster_manager.py

```diff
@@ -25,9 +25,9 @@
 from typing import Tuple
 
 from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.runners.interactive import interactive_beam as ib
```

Comment:
Could you please use the ie.current_env().clusters instead of ib.clusters in interactive internal modules? This could avoid the dependency on interactive_beam modules and avoid potential dependency loop in the future.